src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -98,6 +98,10 @@ export class CalcitePickListItem {
    */
   @Prop({ reflect: true }) value!: string;
 
+  @Watch("value") valueWatchHandler(newValue, oldValue) {
+    this.calciteListItemValueUpdated.emit({ oldValue, newValue });
```

Comment:
Should this be named `calciteListItemValueChange`? Events shouldn't be in past tense and we've been using `change` for events.